### PR TITLE
Add Create goal service

### LINKED ISSUE #6: Add goal creator service
Add the ROS service file to the frida navigation interfaces package

### PR DESCRIPTION
**Pull request by Diego-HC**
Closes #6

### CODE REVIEW

**Review by afr2903 — CHANGES_REQUESTED** [review]
Seems suspicious 🧐

File: frida_navigation_interfaces/srv/CreateGoal.srv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+geometry_msgs/PoseStamped target
+std_msgs/Float64 offset
```
Comment (EmilianoHFlores) [inline frida_navigation_interfaces/srv/CreateGoal.srv]: For use in external scripts add more description to the offset and its direction

**Comment by afr2903** [comment]
The import is missing in the `CMakeLists.txt` file

**Review by EmilianoHFlores — APPROVED** [review]
LGTM

File: frida_navigation_interfaces/CMakeLists.txt [hunk 2]
```diff
@@ -23,6 +24,7 @@ add_action_files(
 add_service_files(
   FILES
   ViewAngle.srv
+  CreateGoal.srv
```
Comment (afr2903) [inline frida_navigation_interfaces/CMakeLists.txt:27]: :+1:

**Review by afr2903 — APPROVED** [review]

**Pull request merged**